torch/onnx/utils.py

```diff
@@ -1132,9 +1132,7 @@ def _run_symbolic_function(g, block, n, inputs, env, operator_export_type=Operat
                         torch._C._jit_pass_onnx_node_shape_type_inference(new_node, _params_dict, opset_version)
                     return new_op_outputs
             else:
-                symbolic_name = "prim_" + op_name
-                domain = ""
-                symbolic_fn = _find_symbolic_in_registry(domain, symbolic_name, opset_version,
+                symbolic_fn = _find_symbolic_in_registry("prim", op_name, opset_version,
                                                          operator_export_type)
                 if symbolic_fn is None:
                     return None
```

Comment:
GitHub won't let me comment on the appropriate line, but below can you please change from single to double quotes?
`if op_name == 'PythonOp':` -> `if op_name == "PythonOp":`